ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -610,6 +610,7 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
       type_constraint_map["Transpose"]["T"] = NGraphDTypes();
       type_constraint_map["Transpose"]["Tperm"] = NGraphIndexDTypes();
       type_constraint_map["Unpack"]["T"] = NGraphDTypes();
+      type_constraint_map["Where"]["T"] = NGraphDTypes();
```

Comment:
right now input type of "where" is bool only, right?
maybe we can say `type_constraint_map["Where"]["T"] = {DT_BOOL}`

Reply:
Yes, bool only, will change